### PR TITLE
IEP-527: Help button added to the status bar

### PR DESCRIPTION
**Pull request by alirana01**

### CODE REVIEW

**Comment by kolipakakondal** [comment]
Hi @alirana01 Could you address the PR comments

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/handlers/HelpContribItemHandler.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+public class HelpContribItemHandler extends AbstractHandler
+{
+
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException
+	{
+		String url = "https://www.esp32.com/viewforum.php?f=40"; //$NON-NLS-1$
+		try
+		{
+			Desktop.getDesktop().browse(new URL(url).toURI());
```
[the remark this answers is not in the document]
Comment (alirana01) [reply bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/handlers/HelpContribItemHandler.java]: Updated
image

**Comment by alirana01** [comment]
image
Screenshot

**Review by kolipakakondal — APPROVED** [review]
LGTM

**Pull request merged**